web/files/referencedby.go

```diff
@@ -83,5 +83,5 @@ func RemoveReferencedHandler(c echo.Context) error {
 		return wrapVfsError(err)
 	}
 
-	return c.NoContent(http.StatusNoContent)
+	return c.NoContent(http.StatusOK)
```

Comment:
Why this change? 204 No Content is a good status code to indicate the success of a deletion (body is empty)